src/components/Schedule.js

```diff
@@ -14,84 +26,90 @@ const Schedule = () => {
-                              style={{ backgroundColor: event.theme }}
-                            />
-                            {/* TODO: Need to change hardcoded width here */}
-                            <td width="50px">
-                              <div>
-                                <img src={Arrow} alt="arrow icon" />
-                              </div>
-                            </td>
-                            {/* TODO: Need to change hardcoded width here */}
-                            <td width="180px">
-                              <div className="category">
-                                <p className="category__time">{event.time}</p>
-                                <p
-                                  className="category__type"
-                                  style={{ color: event.theme }}
-                                >
-                                  {event.type}
-                                </p>
-                              </div>
-                            </td>
-                            {/* TODO: Need to change hardcoded width here */}
-                            <td width="450px">
-                              <div className="location">
-                                <p className="location__title">{event.title}</p>
-                                <a
-                                  className="location__zoom"
-                                  href={event.location}
-                                >
-                                  {event.location}
-                                </a>
-                              </div>
-                            </td>
-                            {/* TODO: Need to change hardcoded width here */}
-                            <td width="300px">
-                              <div className="audience">
-                                <p
-                                  className={
-                                    event.audience ? "audience__type" : ""
-                                  }
-                                >
-                                  {event.audience}
-                                </p>
-                                <p
-                                  className={
-                                    event.company ? "audience__company" : ""
-                                  }
-                                >
-                                  {event.company}
-                                </p>
-                              </div>
-                            </td>
-                            <td>
-                              <img src={CalendarIcon} alt="calendar icon" />
-                            </td>
-                          </tr>
-                        );
-                      })}
-                    </tbody>
-                  </Table>
-                </Tab>
-              );
-            })}
-          </Tabs>
-        </Col>
-      </Row>
```

Comment:
I have some changes here but idk why it just reformatted everything...

Reply:
u can view with ignore whitespace changes! in the little gear icon

Reply:
wait where is the gear icon?

Reply:
![Screen Shot 2021-01-21 at 4 37 57 PM](https://user-images.githubusercontent.com/22948432/105415688-14e4d680-5c07-11eb-8dfa-ee3e53cf2560.png)

Reply:
ooooo

Reply:
yeee!